assets/src/settings-page/settings-footer.js

```diff
@@ -2,33 +2,43 @@
  * WordPress dependencies
  */
 import { Button } from '@wordpress/components';
-import { useContext, useEffect } from '@wordpress/element';
+import { useContext, useEffect, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { Options } from '../components/options-context-provider';
 import { ReaderThemes } from '../components/reader-themes-context-provider';
+import { AMPNotice, NOTICE_TYPE_SUCCESS } from '../components/amp-notice';
 
 /**
  * The bottom section of the settings page.
  */
 export function SettingsFooter() {
 	const { didSaveOptions, editedOptions, saveOptions, savingOptions } = useContext( Options );
 	const { downloadingTheme } = useContext( ReaderThemes );
+	const [ saved, setSaved ] = useState( false );
 
 	const { reader_theme: readerTheme, theme_support: themeSupport } = editedOptions;
 
 	/**
-	 * Submits the PHP-generated form on the page after options have saved via REST.
-	 *
-	 * @todo Eliminate this once the Plugin Suppression logic is moved to use the REST API.
+	 * Shows a saved notice on success.
 	 */
 	useEffect( () => {
 		if ( true === didSaveOptions && ! downloadingTheme ) {
-			document.querySelector( 'form#amp-settings' ).submit();
+			setSaved( true );
+
+			const timeout = setTimeout( () => [
+				setSaved( false ),
+			], 9000 );
+
+			return () => {
+				clearTimeout( timeout );
+			};
 		}
+
+		return () => undefined;
 	}, [ didSaveOptions, downloadingTheme ] );
 
 	const disabled = ! themeSupport ||
```

Comment:
I just noticed that this needs to be updated to account for the initial state. 

When I first access the settings screen, I see:

![image](https://user-images.githubusercontent.com/134745/87601078-3342a600-c6a9-11ea-853c-7aa90a870715.png)

When I expected to see the disabled state:

![image](https://user-images.githubusercontent.com/134745/87601118-43f31c00-c6a9-11ea-834d-63babe925e80.png)

Aside: Let's make the gray darker because it has poor contrast for a11y.

Reply:
Much better! 

![image](https://user-images.githubusercontent.com/134745/87607287-d8617c80-c6b1-11ea-99bc-35fe0d36b886.png)